src/swagger.ts

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+import * as vscode from 'vscode';
+
+export class Swagger {
+    constructor(definition: any, location: string) {
+        this.swagger = definition;
+        this.location = location;
+    }
+
+    private swagger: any = null;
+    private location: string = "";
+
+    public getExampleNames(path: string, method: string): string[] {
+        let examples: string[] = [];
+        let __this = this;
+
+        for (var name in this.swagger.paths[path][method]['x-ms-examples']) {
+            examples.push(__this.swagger.paths[path][method]['x-ms-examples'][name]['$ref']);
+        }
+
+        return examples;
+    }
+
+    public generateRestApiTasks(path: string, method: string, example: any): string {
+                        
+        let playbook: string = ""
+
+        let url: string = /*this.swagger['schemes'][0] + '://' + this.swagger['host'] + '/' +*/ path;
+        url = url.replace("{subscriptionId}", "{{ lookup('env','AZURE_SUBSCRIPTION_ID') }}");
+
+        method = method.toLowerCase();
+        playbook += //"# https://docs.microsoft.com/en-us/rest/api/" + area + "/" + resource + "/" + internalMethod + '\r';
+                    "- name: Call REST API - " + this.swagger.paths[path][method]['operationId'] + "\r" +
+                    "\t" + ((method == 'get') ? "azure_rm_resource_facts" : "azure_rm_resource") + ":\r" +
```

Comment:
pls replace \t  with whitespace array constants

Reply:
playbookManager is replacing tabs with spaces but it doesnt' really matter

Reply:
of course they matter, and they are replaced later :-)

Reply:
this is a public api.  you really want your user do tab replace by themselves?

Reply:
I believe they are replaced automatically when inserting sample using code in playbookManager. I could recheck it again...

Reply:
edit.insert(new vscode.Position(0, 0)….. in vscode works in such a way that all inserted tabs are replaced with spaces according to settings appropriate for current editor